www/docs/en/dev/plugin_ref/spec.md

```diff
@@ -17,700 +17,498 @@ license: >
     specific language governing permissions and limitations
     under the License.
 
-title: Plugin Specification
+title: Plugin.xml
 ---
 
-# Plugin Specification
-
-The `plugin.xml` file is an XML document in the `plugins` namespace:
-`http://apache.org/cordova/ns/plugins/1.0`. It contains a top-level
-`plugin` element that defines the plugin, and children that define the
-structure of the plugin.
-
-A sample plugin element:
+# Plugin.xml
 
-    <?xml version="1.0" encoding="UTF-8"?>
-    <plugin xmlns="http://apache.org/cordova/ns/plugins/1.0"
-        xmlns:android="http://schemas.android.com/apk/res/android"
-        id="com.alunny.foo"
-        version="1.0.2">
+Plugin.xml file defines the structure and settings required for your plugin. It has several elements to provide details about your plugin.
 
-## _plugin_ Element
+## plugin
 
-The `plugin` element is the plugin manifest's top-level element. It
-features the following attributes:
+  The `plugin` element is the plugin manifest's top-level element.
 
-* `xmlns` (required):
-  The plugin namespace, `http://apache.org/cordova/ns/plugins/1.0`. If
-  the document contains XML from other namespaces, such as tags to be
-  added to the `AndroidManifest.xml` file, those namespaces should
-  also be included in the top-level element.
+  Attributes(type) | Description
+  ---------------- | ------------
+  xmlns(string) | *Required* <br/> The plugin namespace, `http://apache.org/cordova/ns/plugins/1.0`. If the document contains XML from other namespaces, such as tags to be added to the `AndroidManifest.xml` file in the case of Android, those namespaces should also be included in the <plugin> element.
+  id(string) | *Required* <br/> A npm-style identifier for the plugin.
+  version(string) | *Required* <br/> A version number for the plugin, that matches the following major-minor-patch style regular expression: `^\d+[.]\d+[.]\d+$`
```

Comment:
Minor thing: We actually support slightly more complex version numbers. For example `2.1.1-dev` (https://github.com/apache/cordova-plugin-camera/blob/master/plugin.xml#L25). I'm going to guess that we probably support the specification at http://semver.org/